portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+/** Object that stores comment data. */
+public class Comment {
+  String text;
+  String commenterName;
+  String commenterEmail;
```

Comment:
Try to follow alphabetic order to declare variables.

Reply:
Done.